server/supernova/dsp_thread_queue/dsp_thread_queue.hpp

```diff
@@ -425,7 +425,7 @@ template <typename runnable, typename Alloc = std::allocator<void*>> class dsp_q
         const int backoff_iterations = 100;
 
         vector<nanoseconds> measured_values;
-        generate_n(back_inserter(measured_values), 16, [] {
+        generate_n(back_inserter(measured_values), 16, [&] {
```

Comment:
Our style guide mentions using explicit capture where possible (https://github.com/supercollider/supercollider/wiki/Cpp-code-style-guidelines#lambdas)

So this becomes (at a quick glance - I might have missed something):

```diff
- generate_n(back_inserter(measured_values), 16, [&] {
+ generate_n(back_inserter(measured_values), 16, [&max_backup_loops, &watchdog_iterations] {
```

Reply:
`max_backup_loops` does not need to be captured because it's a static member. i'm pretty sure the C++ standard dictates that if `backoff_iterations` is constexpr then a lambda can read its value without capturing it, but apparently MSVC disagrees (https://godbolt.org/z/v7D-2s).

in general in C++ you want to pass small data (things that can fit in 1 or 2 registers, i.e. up to 128 bits on x86_64) by value, and everything larger by reference. this (generally) produces faster and smaller code. a copy is cheap for small types and you save time/instructions by referencing memory directly. it doesn't actually make a difference here AFAIK, but i would write this capture list as `[backoff_iterations]` (copying the value) out of hygienic habit.